drivers/eeprom/CMakeLists.txt

```diff
@@ -13,3 +13,5 @@ zephyr_library_sources_ifdef(CONFIG_EEPROM_EMULATOR eeprom_emulator.c)
 zephyr_library_sources_ifdef(CONFIG_EEPROM_TMP116 eeprom_tmp116.c)
 zephyr_library_sources_ifdef(CONFIG_EEPROM_XEC eeprom_mchp_xec.c)
 zephyr_library_sources_ifdef(CONFIG_EEPROM_FAKE eeprom_fake.c)
+
+zephyr_library_sources_ifdef(CONFIG_EMUL_EEPROM_AT2X emul_atmel_at24.c)
```

Comment:
This breaks the existing naming scheme for EEPROM driver implementation files.

Reply:
This is the current name for the Kconfig so I was trying to do a minimal delta, I went ahead and included a rename and deprecation of the old Kconfig so we can move it to the correct naming scheme.

Reply:
This is still not addressed.

Reply:
Sorry, I thought the issue was the Kconfig.